.github/workflows/build-and-upload-staging.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           tags: ${{ env.ECR_OPERATOR_STAGING_IMAGE }}
           platforms: linux/amd64, linux/arm64
 
-  e2e-test:
+  java-e2e-test:
```

Comment:
can we have a single e2e workflow that has both java and python? We are using our e2e testing as part of our release workflow as well which will make it easier for us to run our whole e2e suite

Reply:
Yes, this is a choice too. We can plan on merging the two tests into one workflow. But one concern is if we want to monitor the java & python test separately,  which might be more convenience for problem detection and debugging?

Reply:
Updated to use a new workflow calling the both workflows.